e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -352,6 +352,40 @@ describe("scenarios > dashboard", () => {
           .and("contain", "18,760");
       });
 
+      it("should save changes to a dashboard after using the 'Add a chart' button from an empty tab (metabase#53132)", () => {
+        cy.log("add an existing card");
+        cy.findAllByTestId("dashboard-header").icon("pencil").click();
+        cy.findAllByTestId("dashboard-header").icon("add").click();
```

Comment:
Did you mean `cy.findByTestId()` or do we actually have multiple elements with this same test id? Sounds unlikely but I wouldn't be surprised even if we did.

Reply:
ooops, i suppose bad auto-complete. good catch.